src/lib.rs

```diff
@@ -79,24 +79,14 @@
 //! via `endian_fd`
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(all(feature = "alloc", not(feature = "std")), feature(alloc))]
 
 #[cfg(feature = "std")]
 extern crate core;
 
-#[cfg(all(feature = "alloc", not(feature = "std")))]
+#[cfg(feature = "alloc")]
 #[macro_use]
 extern crate alloc;
```

Comment:
So this is like a “magical” external crate now, similar to core ? I just realized we had this before. And I don’t really know how it works haha

So In 1.36 what changer is that we can use this now without the alloc mod simulating it below for std targets ? (I guess stabilization means it’s available for both no_std and std configurations ?)

I should probably just read the rfc. 

Anyway thanks for the refactor !

Reply:
Yes, it's the same as core, and it's always available, so we can use it all the time even for std builds on stable rust.